src/applications/proxy-rewrite/README.md

```diff
@@ -1,15 +1,22 @@
 # Proxy Rewrite (AKA Injected header/footer support)
-This `proxy-rewrite` application is used to inject site-wide VA.gov components into webpages and domains outside of those in our domain or in our source code. The affected sites are generally referred to as "TeamSite", because TeamSite is the name of the CMS in use on those pages.
```

Comment:
Changes here describe how the code has changed and rearrange info a bit for priority.